conf.py

```diff
@@ -32,6 +32,14 @@
 else:
     acos_submit_base_url = 'https://acos.cs.aalto.fi'
 
+override = {
+    'jututfeedback': {
+        'url': ('http://jutut:8082/feedback/testcoursemultilang/{key}' if on_container
+                else 'https://jutut.minus.cs.aalto.fi/feedback/testcoursemultilang/{key}'),
```

Comment:
`jutut.minus.cs.aalto.fi`: I will change this URL too since now it refers to a testing environment.